core/src/conn.rs

```diff
@@ -309,7 +319,7 @@ impl MongoConnection {
     }
 
     /// Gets the ADF version the client is connected to.
-    pub fn get_adf_version(&self) -> Result<String> {
+    pub fn get_server_version(&self) -> Result<String> {
```

Comment:
renamed as this was still being used in `functions` and failed against an atlas cluster since datalake wasn't there